### PR TITLE
fix(dropdown): z-index, open/close + uuid

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Fixing issues in the dropdown component.

1. Dropdown z-index of menu items.
2. Dropdown not working when placed in the sdds-modal component

**Solving issue**  
Fixes: Bug reported through Teams channel and in a video call.

**How to test**  
1. Start the development environment from the root
3. run linking command 
4. run linking command in Angular12 demo example
5. run Angular12 example
6. See if dropdown menus looks broken when two dropdowns are close to each other
7. See if dropdown works when placed in the modal component

[DOCS: How to link](https://scaniaazureservices-my.sharepoint.com/:o:/r/personal/nikola_johnny_mirkovic_scania_com/Documents/Notebooks/IXCD%20Book?d=wa2c3d5d9d0dc4cb9bee55e8a81cedb1b&csf=1&web=1&e=Fhq7wB)

### CODE REVIEW

**Comment by theJohnnyMe** [comment]
[4.4.3-beta.0](https://www.npmjs.com/package/@scania/components/v/4.4.3-beta.0) release confirmed that this one fixes issues reported in "Development" channel in Teams.

File: components/src/components/dropdown/dropdown.scss [hunk 1]
```diff
@@ -147,13 +148,13 @@
     .sdds-dropdown-menu {
       top: auto;
       bottom: 100%;
-      box-shadow: 0 -1px 3px 0 rgba(0, 0, 0, 0.1);
+      box-shadow: 0 -1px 3px 0 rgb(0 0 0 / 10%);
```
Comment (gaonita) [inline components/src/components/dropdown/dropdown.scss:151]: neat way of writing color with opacity🙌🏻

**Review by gaonita — APPROVED** [review]
LGTM☀️

File: components/src/components/dropdown/dropdown.scss (hunk 1 above)
Comment (theJohnnyMe) [reply components/src/components/dropdown/dropdown.scss:151]: Thanks, I think one of the linters did it actually and I like it too! 😃

**Pull request merged**